From ce56a6f924274a859db9bb46a8181217d9021c74 Mon Sep 17 00:00:00 2001
From: Matthieu Darbois <mayeut@users.noreply.github.com>
Date: Sat, 13 May 2023 06:14:32 +0200
Subject: [PATCH 55/79] Fix test failure introduced in
 51af4082cc898b122b88f11fd34033fc00fad81e (#145)

modules/arch/x86/tests/x86_test.sh failed after 51af4082cc898b122b88f11fd34033fc00fad81e commit.
This fixes this test.
---
 libyasm/errwarn.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/libyasm/errwarn.c b/libyasm/errwarn.c
index 18958db9..983e0819 100644
--- a/libyasm/errwarn.c
+++ b/libyasm/errwarn.c
@@ -111,11 +111,11 @@ void
 yasm_errwarn_initialize(void)
 {
     /* Default enabled warnings.  See errwarn.h for a list. */
-    warn_class_enabled = 
+    warn_class_enabled =
         (1UL<<YASM_WARN_GENERAL) | (1UL<<YASM_WARN_UNREC_CHAR) |
         (1UL<<YASM_WARN_PREPROC) | (0UL<<YASM_WARN_ORPHAN_LABEL) |
         (1UL<<YASM_WARN_UNINIT_CONTENTS) | (0UL<<YASM_WARN_SIZE_OVERRIDE) |
-        (1UL<<YASM_WARN_IMPLICIT_SIZE_OVERRIDE);
+        (1UL<<YASM_WARN_IMPLICIT_SIZE_OVERRIDE) | (0UL<<YASM_WARN_SEGREG_IN_64BIT);
 
     yasm_eclass = YASM_ERROR_NONE;
     yasm_estr = NULL;
@@ -217,7 +217,7 @@ errwarn_data_new(yasm_errwarns *errwarns, unsigned long line,
     }
 
     if (replace_parser_error && ins_we && ins_we->type == WE_PARSERERROR) {
-        /* overwrite last error */      
+        /* overwrite last error */
         we = ins_we;
     } else {
         /* add a new error */
-- 
2.49.0

